documentation/src/main/asciidoc/topics/upgrading.adoc

```diff
@@ -1,5 +1,10 @@
 = Upgrading from 12.0 to 12.1
 
+== Search
```

Comment:
should be a new section "Upgrading from 12.1 to 13.0"

Reply:
Yes, I think I've sent the PR before 13.0.0-SNAPSHOT